pkg/controllers/dashboard/systemcharts/controller.go

```diff
@@ -49,7 +49,7 @@ const (
 var (
 	primaryImages = map[string]string{
 		chart.WebhookChartName:           "rancher/rancher-webhook",
-		chart.ProvisioningCAPIChartName:  "rancher/mirrored-cluster-api-controller",
+		chart.ProvisioningCAPIChartName:  "registry.k8s.io/cluster-api/cluster-api-controller",
```

Comment:
Can you confirm that we decided to use the upstream image instead of the morrired one?

Reply:
OK, thanks for the confirmation. Let's leave this comment open as a reminder until the change is reverted.